src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -0,0 +1,318 @@
+swagger: '2.0'
+info:
+  description: |
+    **Model API**
+
+    Provides access to the model features of Alfresco Content Services.
+  version: '1'
+  title: Alfresco Content Services REST API
+basePath: /alfresco/api/-default-/public/alfresco/versions/1
+securityDefinitions:
+  basicAuth:
+    type: basic
+    description: HTTP Basic Authentication
+security:
+  - basicAuth: []
+consumes:
+  - application/json
+produces:
+  - application/json
+tags:
+  - name: aspects
+    description: Retrieve and manage aspects
+
+parameters:
+  aspectIdParam:
+    name: aspectId
+    in: path
+    description: The Qname of a aspect.
+    required: true
+    type: string
+
+paths:
+  '/aspects':
+    get:
+      tags:
```

Comment:
Although we still don't use it, it would be good to add the x-alfresco-since: "7.0" property for future use.